### PR TITLE
Start game materials page

### PR DESCRIPTION
**Pull request by nirs**
@rollandf @sleviim please review and add content

### CODE REVIEW

File: docs/game.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+### Run example driver
+
+## Writing your driver
+
+### Writing your own None driver
+
+### Print your car location
+
+### Print the obstacle a head
+
```
Comment (sleviim) [inline docs/game.md]: '''
obstacle = world.get((x, y - 1))
print obstacle
'''

**Comment by nirs** [comment]
@sleviim is this good enough to merge in the current state, so you can work on it?

File: docs/game.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+### Restarting after exceptions
+
```
Comment (sleviim) [inline docs/game.md]: Re-run your driver:
'''
./rose-client yourname.py
'''

File: docs/game.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+### Driving in other car lane
+
+### Remembering your lane
+
+### Looking more than one penguin a head
+
+### Running against other drivers
+
+- Finding ip address
+- Connecting rose-client to remote machine
```
Comment (sleviim) [inline docs/game.md]: For example, in order to compete with client foo.py on 10.35.35.35 run:
'''
./rose-client -s 10.35.35.35 foo.py
'''

File: docs/game.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+### Driving in other car lane
+
+### Remembering your lane
+
+### Looking more than one penguin a head
+
+### Running against other drivers
+
+- Finding ip address
```
Comment (sleviim) [inline docs/game.md]: Finding your IP address
'''
ip addr
'''
Look for an address that starts with 10.35

Comment (nirs) [reply docs/game.md]: @sleviim I'm merging this so you can post these changes in a new PR.

**Pull request merged**